.github/workflows/publish.yml

```diff
@@ -69,6 +69,8 @@ jobs:
         run: |
           tox -e docs
           touch build/sphinx/html/.nojekyll  # allow underscores in URL path
+          # remove unnecessary build files
+          sudo rm -rf build/sphinx/html/.doctrees
```

Comment:
Could we achieve the same effect by updating `.gitignore` file with appropriate entry?

Reply:
I tried it initially for iqm-client ([commit 5de3df2](https://github.com/jkotilahti/iqm-client/commit/332531eafafd23c81f5b47e101894364b70a4d59) in https://github.com/jkotilahti/iqm-client/commits/main/), but it didn't work.